proto/axelar/evm/v1beta1/events.proto

```diff
@@ -109,6 +111,36 @@ message ConfirmGatewayTxStarted {
       [ (gogoproto.nullable) = false, (gogoproto.embed) = true ];
 }
 
+message PollMapping {
+  bytes tx_id = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customtype) = "Hash",
+    (gogoproto.customname) = "TxID"
+  ];
+  uint64 poll_id = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customname) = "PollID",
+    (gogoproto.customtype) =
+        "github.com/axelarnetwork/axelar-core/x/vote/exported.PollID"
+  ];
+}
+
+message ConfirmGatewayTxsStarted {
+  repeated PollMapping poll_mappings = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "poll_mappings,omitempty"
```

Comment:
It's a non-nullable slice, our event tests failing as creating the event from reflection has a null, instead of `[]`. 
The default tag is still `poll_mappings`, the api defined in the way that needs to specify the name first.